01_fib/choohy/main.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+
+	fmt.Fprintln(out, "Call Fibonacci numbers!")
```

Comment:
Little nit, but there is no need to include this `fmt.Println()` here. Please remove this and the additional line above.
Lab stated 
>Call fib(7) in main to print

So just need to call 
```go
func main() {
    fib(7)
}
```